<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>279</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Trellix Desatendido  By: Andrés</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #343338</string>
  </property>
  <widget class="QWidget" name="Ventana">
   <widget class="QLabel" name="Lbl_Estado">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>741</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
font-family: Arial;
color: #edeef0;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>751</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>26</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    background-color: #8A8B8F; /* Color de fondo gris claro */
    border: 2px solid #8A8B8F; /* Borde gris */
    border-radius: 10px; /* Esquinas redondeadas */
    height: 20px; /* Altura de la barra */
    text-align: center; /* Texto centrado */
}

QProgressBar::chunk {
    background-color: #FFFFFF; /* Color de progreso blanco */
    margin: 0px; /* Sin margen interno */
    border-radius: 8px; /* Esquinas redondeadas para los 'chunks' */
}

QProgressBar::text {
    color: #edeef0; /* Color de texto negro */
    /* Si se requiere, se puede agregar más estilización al texto aquí */
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="Lbl_N_Estado">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
font-family: Arial;
color: #edeef0;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
